ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-oadp-1.4__4.12.yaml

```diff
@@ -1,14 +1,12 @@
 build_root:
   image_stream_tag:
     name: builder
-    namespace: ocp
-    tag: rhel-9-golang-1.22-openshift-4.17
+    namespace: konveyor
+    tag: ubi9-v1.22.2
 images:
 - dockerfile_path: Dockerfile
-  from: src
```

Comment:
these changes are needed?

when we change back to ocp builder image, do we need to add them back?

Reply:
Essentially when from: statement is removed, the first or last (can't remember which) `FROM` line in Dockerfile will be used. Otherwise when from: src is used, FROM is replaced with `base_images` at the top of this yaml.

Reply:
They were needed prior to mirroring konveyor/builder to be made available as a base_images. But mirroring is still required here for unit tests to run on.. so actually right now it isn't required.

Reply:
> when we change back to ocp builder image, do we need to add them back?

If we want to switch back to ocp builder image, yes we add it back.

Reply:
better to left commented out? (do not know if `make jobs` and other commands remove comments)